svgdigitizer/entrypoint.py

```diff
@@ -306,16 +363,26 @@ def digitize_cv(
         metadata = yaml.load(metadata, Loader=yaml.SafeLoader)
 
     with open(svg, mode="rb") as infile:
-        cv = CV(
+        svgfigure = CV(
             _create_svgplot(infile, sampling_interval=sampling_interval, skewed=skewed),
             metadata=metadata,
             force_si_units=si_units,
         )
 
+    _create_outfiles(
+        svgfigure=svgfigure, svg=svg, outdir=outdir, bibliography=bibliography
+    )
+
+
+def _create_outfiles(svgfigure, svg, outdir, bibliography):
```

Comment:
@saraedum I added a new helper function, to reduce duplicate code in the different digitization commands in the CLI. I guess this is fine.